docs/rollout/in-product-onboarding.md

```diff
@@ -0,0 +1,32 @@
+---
+title: In-product Onboarding
+layout: rollout
+---
+
+## Prefer in-product onboarding
+
+Tilt is a powerful tool, and thus requires some investment on the part of a new user to learn the basics. We recommend providing a [transition guide](../rollout/introduce-tilt) to help developers with onboarding. Ideally, many of the onboarding steps can be driven by _in-product onboarding_ instead, where Tilt itself guides the new user through initial set up, and usage of features, from basic to advanced.
+
+## Why it's effective
+
+- With in-product onboarding, you don't need to maintain a separate onboarding doc, where steps can get out of date over time. The source of truth is the product itself.
+- The system keeps track of where you are, as the user, during the onboarding process, and provides customized help and reminders. If you do onboarding over several sittings, you immediately pick up where you left off at the beginning of each session, since the system saves state.
+- As the DevEx person, the system provides you real-time state of where each user is in their individual onboarding, as well as metrics of how past users performed at certain steps. You see bottlenecks and improve onboarding over time from this insight.
+
+## Future enhancement
+
+Tilt doesn't have in-product onboarding yet. If this sounds interesting to you, [schedule a chat](https://calendly.com/dbentley/tilt-enterprise) and let us know which parts of onboarding you'd want to see in Tilt.
+
+## Custom scripts
+
+In the meantime, consider using offering custom scripts to help a new user install the preferred [local Kubernetes cluster](../choosing_clusters.html) or [install](../install.html)/[upgrade](../upgrade) the preferred version of Tilt. You can also take advantage of [Tiltfiles being written in the Python-dialect of Starklark](../tiltfile_concepts) to create custom logic to help new users when the first run Tilt.
```

Comment:
I'd switch the order of Custom Scripts and Future Enhancements.

I think it's possible to do an alright job of this using local_resource, and I'd love to explore it with a team. So maybe also expand it to mention that you could do something and we'd pair with you or some such?

Reply:
Done.